statusboard/src/components/ProjectsTable/ColumnHeader.scss

```diff
@@ -47,3 +47,18 @@
     }
   }
 }
+
+@media screen and (max-width:800px) {
+  .column-header__contents {
+      display: flex;
+      padding-right: 2rem;
+      align-items: stretch;
+      width:95%;
```

Comment:
Why do you want the width to be 95%? Is it to reduce the border?
The reason I ask is that I think it might cause the scroll bar to be triggered easier.

Reply:
It's because each header has padding built in so setting width to 100% actually causes the scrollbar to appear. I don't want to remove the padding (and actually think the whole header situation could use a bit more padding), so this was a workaround. I've changed to 100%-2rem to be more specific.

Reply:
I think you are overestimating my CSS knowledge 😛 

I don't mind the padding, nor more padding or margin to make it easier to separate elements visually. What I don't understand is why we are setting the width in the first place and what its purpose is in the code?

If I understand it correctly we are trying to make the `column-header__contents` div take up 95% of the width of the table header, which in turn is 40% of the table row width.

Does it make things easier or worse if we change

https://github.com/codeforamerica/brigade-project-index-statusboard/blob/facbc2f652b7b6d026a4fb55342b093968d2cf52/statusboard/src/components/ProjectsTable/ProjectsTable.scss#L23-L25

to be

```
@media screen and (min-width:800px) {
	th:nth-of-type(2) {
	    width: 40%;
	}
}
```

And instead of setting the width of the `input` you could set a margin on `.text-input`.